app/tests/spec/models/flow.js

```diff
@@ -0,0 +1,205 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define(function (require, exports, module) {
+  'use strict';
+
+  var assert = require('chai').assert;
+  var AuthErrors = require('lib/auth-errors');
+  var Flow = require('models/flow');
+  var ResumeToken = require('models/resume-token');
+  var sinon = require('sinon');
+  var Url = require('lib/url');
+  var WindowMock = require('../../mocks/window');
+
+  var FLOW_ID = 'F1031DF1031DF1031DF1031DF1031DF1031DF1031DF1031DF1031DF1031DF103';
```

Comment:
Would these names be easier to understand if one were

`BODY_FLOW_ID` and one were `RESUME_TOKEN_FLOW_ID` or something similar to indicate its source?